circle.yml

```diff
@@ -1,9 +1,22 @@
 machine:
-  python:
-    version: 3.5.1
-  environment:
-    DB_NAME: circle_test
-    DB_USER: ubuntu
-    DB_HOST: 127.0.0.1
-  services:
-    - postgresql
+    python:
+        version: 3.5.1
+    environment:
+        DB_NAME: circle_test
+        DB_USER: ubuntu
+        DB_HOST: 127.0.0.1
+    services:
+        - postgresql
+        - docker
+
+dependencies:
+    post:
+        - docker build -t $AWS_ECR_ENDPOINT.dkr.ecr.us-east-1.amazonaws.com/cognoma-task-service:$CIRCLE_SHA1 .
+
+deployment:
+    production:
+        branch: master
+        commands:
+            - chmod +x ecr_push.sh ecs_deploy.sh
+            - ./ecr_push.sh
+            - ./ecs_deploy.sh -t 180 -c $AWS_CLUSTER_NAME -n $AWS_SERVICE_NAME -i $AWS_ECR_ENDPOINT.dkr.ecr.us-east-1.amazonaws.com/cognoma-task-service:$CIRCLE_SHA1
```

Comment:
I recommend long args whenever you're writing shell scrips (i.e. not just typing in command line), since they're more explicit and readable. Also consider splitting long command onto multiple lines, which will help with git diff later.